CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,2 @@
-Fix: check access to active attribute of device before use it (twin of iota-json#556)
+Fix: check access to active attribute of device before use it (twin of iota-json#576)
```

Comment:
```suggestion
Fix: check access to active attribute of device before use it (twin of iota-json#578)
```

Changed to the real twin PR ( iota-json#576 was an issue about other topic in which we detect the problem)

Reply:
As discussed in other channels, NTC